plugins/src/main/java/opengrok/auth/plugin/LdapUserPlugin.java

```diff
@@ -157,8 +158,7 @@ public void fillSession(HttpServletRequest req, User user) {
         String dn = null;
```

Comment:
the user is still set to null (if that is something you tried to avoid based on your description) liine 151

Reply:
As far as I remember, this was a problem when I tried to use the `null` `LdapUser` session attribute as a indication of negative caching. Otherwise both `LdapAttrPlugin` and `LdapFilterPlugin` deal with it just fine.